client/blocks/qr-code-login/jetpack.jsx

```diff
@@ -0,0 +1,52 @@
+import { ExternalLink } from '@automattic/components';
+import { translate } from 'i18n-calypso';
+import { TokenQRCode } from './index';
+
+const steps = [
+	// translation: Link to the Jetpack App.
+	translate( 'Open the {{link}}%(name)s App{{/link}} on your phone.', {
+		args: {
+			name: 'Jetpack',
+		},
+		components: {
+			link: <ExternalLink target="_blank" href="https://jetpack.com/app?campaign=login-qr-code" />,
+		},
+	} ),
+	translate( 'Tap the Me tab.' ),
+	translate( 'Tap the Scan Login Code option.' ),
+	translate( 'Point your phone to the following QR code to scan it.' ),
+];
+
+const notice = translate(
+	"Logging in via the Jetpack app is {{strong}}not available{{/strong}} if you've enabled two-step authentication on your WordPress.com account.",
+	{
+		components: {
+			strong: <strong />,
+		},
+	}
+);
```

Comment:
It's better to use `useTranslate` instead of this approach. The problem with using `translate` outside of components is that, if the user switches languages or the translation file is loaded async after rendering, the component wouldn't reflect the changes.

Reply:
Thanks, I fixed that in 6d9189a